src/pypromice/process/get_l3.py

```diff
@@ -1,46 +1,108 @@
 #!/usr/bin/env python
-import logging, os, sys, unittest
+import os, unittest, pkg_resources
+import pandas as pd
+import numpy as np
+import xarray as xr
 from argparse import ArgumentParser
-from pypromice.process.aws import AWS
+from pypromice.process import getVars, getMeta, addMeta, getColNames, \
+    roundValues, resampleL2, writeAll
+from pypromice.process.L1toL2 import correctPrecip
+from pypromice.process.L2toL3 import toL3
+from sys import exit
 
-def parse_arguments_l3():
-    parser = ArgumentParser(description="AWS L3 processor")
-
-    parser.add_argument('-c', '--config_file', type=str, required=True,
-                        help='Path to config (TOML) file')
-    parser.add_argument('-i', '--inpath', default='data', type=str, required=True, 
-                        help='Path to input data')
-    parser.add_argument('-o', '--outpath', default=None, type=str, required=False, 
-                        help='Path where to write output')
-    parser.add_argument('-v', '--variables', default=None, type=str, 
-                        required=False, help='File path to variables look-up table')
-    parser.add_argument('-m', '--metadata', default=None, type=str, 
-                        required=False, help='File path to metadata')
-    args = parser.parse_args()
+def parse_arguments_getl3(debug_args=None):
+    parser = ArgumentParser(description="AWS L3 script for the processing L3 data from L2 and merging the L3 data with its historical site. An hourly, daily and monthly L3 data product is outputted to the defined output path")
+    parser.add_argument('-s', '--file1', type=str, required=True, nargs='+',
+                        help='Path to source L2 file')
+    # here will come additional arguments for the merging with historical stations
+    parser.add_argument('-v', '--variables', default=None, type=str, required=False, 
+    			 help='Path to variables look-up table .csv file for variable name retained'''),
+    parser.add_argument('-m', '--metadata', default=None, type=str, required=False, 
+    			 help='Path to metadata table .csv file for metadata information'''),
+    parser.add_argument('-d', '--datatype', default='raw', type=str, required=False, 
+    			 help='Data type to output, raw or tx')
+    args = parser.parse_args(args=debug_args)
+    args.file1 = ' '.join(args.file1)
```

Comment:
Why are you concatenating the list of input filers to a string?

Reply:
see comment above:

> It was the only way I found to feed paths containing spaces to getl3 for testing purpose:
> - allow several input to be given for --file1 (since a path with spaces is automatically split on spaces)
> - concatenate back those pieces afterwards to restore the user-given path for --file1
> 
> Penny removed those options, so any suggestion on how to do this in the current implementation is welcome!